target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -232,6 +232,9 @@ ramips_board_detect() {
 	*"JHR-N926R")
 		name="jhr-n926r"
 		;;
+	*"ZyXEL Keenetic Viva")
+		name="kng_rc"
```

Comment:
I'm not that satisfied with the name kng_rc. I don't see any relation to the full board name.